src/style/_theme-color-variables.scss

```diff
@@ -36,4 +38,23 @@
         --lime-on-surface-color,
         var(--lime-text-primary-on-background-color)
     );
+
+    // Elevated surfaces such as buttons, popovers or menus
+    // Which normally have a drop-shadow effect, need to be
+    // lighter than the background of the context they are in.
+    // This helps the shadow effect be more visible, and improves
+    // end-users' perception of the depth we are trying to create.
+    // In the `dark` color-scheme, it is the same story.
+    // However, that also means that using `--contrast-100` will render
+    // them with the darkest contrast swatch color, in dark mode.
+    // To avoid that, and instead of individually adding a
+    // `@media (prefers-color-scheme: dark)` query to CSS styles of
+    // each elevated surface, we use this CSS variable instead.
+    // Consumers can then tweak this, if they have used the component on a
+    // background color which interferes with this lightened surface,
+    // or provides a poor contrast.
+    --lime-elevated-surface-background-color: rgb(var(--contrast-100));
+    @include mixins.in(dark-mode) {
+        --lime-elevated-surface-background-color: rgb(var(--contrast-200));
```

Comment:
@adrianschmidt this mixin for dark mode will not work. It doesn't since this containing mixin (`theme-color-variables`) is used inside a `:root` in:
https://github.com/Lundalogik/lime-elements/blob/75aed282825b6384dedf616f7e46cc2a65efba9b/src/global/core-styles.scss#L8

and used inside a `:host` in:
https://github.com/Lundalogik/lime-elements/blob/75aed282825b6384dedf616f7e46cc2a65efba9b/src/style/internal/lime-theme.scss#L47

The solution would be that I move this new variable out to another file, for example `core-styles.scss`.

However, I don't think we need to import this mixin in `lime-theme.scss` file generally. The mixin only contains a bunch of CSS variables currently. Maybe hostorically it contained more SCSS variables. Isn't it enough that the mixin is used on `core-styles.scss`? Wouldn't they all be available everywhere via `:root`?